src/test/java/br/com/meli/seu_imovel/service/BiggerRoomTest.java

```diff
@@ -0,0 +1,76 @@
+package br.com.meli.seu_imovel.service;
+
+import br.com.meli.seu_imovel.dto.ComodoDTO;
+import br.com.meli.seu_imovel.dto.ImovelDTO;
+import br.com.meli.seu_imovel.repositories.BairroRepository;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+@SpringBootTest
+@ExtendWith(MockitoExtension.class)
+public class BiggerRoomTest {
+    @Mock
+    private BairroRepository bairroRepository;
+    @InjectMocks
+    private ImoveisService imoveisService;
+
+    @Test
+    @DisplayName("Testa se o maior cômodo retorna com nome correto")
+    public void testeMaiorQuarto(){
+        ImovelDTO imovelDTO = geraImovelTeste();
+        Mockito.when(bairroRepository.findByName("centro")).thenReturn(Optional.of("centro"));
+        ImovelDTO maiorComodo = imoveisService.gerarRelatorio(imovelDTO);
```

Comment:
```suggestion
        ImovelDTO imovelRetorno = imoveisService.gerarRelatorio(imovelDTO);
```

(Opcional) Seria legal trocar esse nome "maiorComodo" para algo mais coerente, tipo "imovelRetorno" ja que é do tipo imovel!

Reply:
Declarar nome que faz sentido né?!
Concordo também, vamos fazer a alteração

Reply:
aisuhdaiushdiuashdiuasd